test/run-xspec-tests.cmd

```diff
@@ -39,12 +39,17 @@ for %%I in (*.xspec) do (
     rem
     rem Run
     rem
-    "%COMSPEC%" /c ..\bin\xspec.bat "%%~I" > "%RESULT_FILE%" 2>&1
+    call :is_schematron %%~nI%

```

Comment:
Just in case, wrapping by `call :is_schematron "%%~nI"` and unwrapping by `set var=%~1` would be safer.